devices/pine64-pinephone/kernel/default.nix

```diff
@@ -22,6 +23,10 @@ mobile-nixos.kernel-builder {
       url = "https://github.com/mobile-nixos/linux/commit/372597b5449b7e21ad59dba0842091f4f1ed34b2.patch";
       sha256 = "1lca3fdmx2wglplp47z2d1030bgcidaf1fhbnfvkfwk3fj3grixc";
     })
+    (fetchurl {
```

Comment:
The explanation for why `fetchpatch` is not used should be added as a comment, to avoid confusion for future contributors.